### PR TITLE
Cancellation Message

### PR DESCRIPTION
**Pull request by jmm8046**

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -193,9 +193,9 @@
           <div class="offset-md-2 col-md-8 ">
             <div class="row">
               <div class="col-12 header-text wrapped-box">
-                <p class="text-left"><span>April 4-5</span></p>
-                <p class="text-left"><span>Penn State University</span></p>
-                <p class="text-left"><span>Business Building</span></p>
+                <p class="text-left"><span>HackPSU Has Been Canceled!</span></p>
```
Comment (stan29308) [inline website/index.html]: Cancelled*?

File: website/index.html [hunk 2]
```diff
@@ -254,21 +254,35 @@
...
-        <div class="header-text">About Us</div>
+          <a name="message">
+        <div class="header-text">A Message to Our Participants</div>
+          </a>
         <div class="header-bar"></div>
       </div>
 
       <div class="col-12 about-text">
+
         <p>
```
Comment (stan29308) [inline website/index.html]: On my phone, but is this tag also supposed to be commented out?

File: website/index.html (hunk 1 above)
Comment (mbullington) [inline website/index.html]: I'm not sure if the exclamation point fits, could you consider removing it?

File: website/index.html (hunk 2 above)
Comment (jmm8046) [reply website/index.html]: which tag? the about-text tag should be there if that's what you mean

File: website/index.html (hunk 1 above)
Comment (jmm8046) [reply website/index.html]: done

Comment (jmm8046) [reply website/index.html]: whoops

**Pull request merged**